docs/02-database.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<p>Wondering if we could reference a document on best practices (using pystac) to do so?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/eoapi-fedgeoday25-workshop/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Yep I that is covered in <a href="https://github.com/developmentseed/eoapi-fedgeoday25-workshop/pull/3" target="_blank">https://github.com/developmentseed/eoapi-fedgeoday25-workshop/pull/3</a></p>